distributed_index_tests/lib/user.py

```diff
@@ -76,7 +112,7 @@ def update_doc(self, doc_id, num_revision=1):
                 if put_resp.status_code == 201:
                     data = put_resp.json()
                     if data["rev"]:
-                        self.docs_info[doc_id] = data["rev"]  # init doc revisions to 0
+                        self.cache[doc_id] = data["rev"]  # init doc revisions to 0
```

Comment:
only quibble..I preferred the earlier name, it was much clearer. Cache not so much.